index.js

```diff
@@ -59,3 +77,5 @@ module.exports = function requireAll(options) {
 function identity(val) {
   return val;
 }
+
+function isObject(arg) { return Object.prototype.toString.call(arg).indexOf('Object') !== -1; }
```

Comment:
Forgive me, as I've never seen a check like this before. Is there something this does over the usual `Object.prototype.toString.call(arg) === '[object Object]'` I think it's emulating?